src/SldStyleParser.ts

```diff
@@ -2248,7 +2251,11 @@ export class SldStyleParser implements StyleParser<string> {
     if (textSymbolizer.placement === 'line') {
       sldTextSymbolizer.push({
         [LabelPlacement]: [{
-          [LinePlacement]: []
+          [LinePlacement]: [{
+            [PerpendicularOffset]: [{
+              '#text': (textSymbolizer as any).perpendicularOffset?.toString()
```

Comment:
PerpendicularOffset exists on LineSymbolizer, but not on textSymbolizer.
However, this should be added on textSymbolizer to support textSymbolizer with `placement: 'line'`, like you do it here, but without the risky `any` cast.